projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/exception/DataJobPropertiesException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class DataJobPropertiesException extends DomainError implements UserFacingError {
+
+  public DataJobPropertiesException(String jobName, String why) {
+    super(
+        String.format("Error processing properties for '%s' data job.", jobName),
```

Comment:
I would write exception instead of error here.

Reply:
fixed.